src/librustc_traits/lowering/environment.rs

```diff
@@ -210,22 +211,23 @@ crate fn environment(tcx: TyCtxt<'_>, def_id: DefId) -> Environment<'_> {
         _ => NodeKind::Other,
     };
 
-    let mut input_tys = FxHashSet::default();
+    // FIXME(eddyb) isn't the unordered nature of this a hazard?
+    let mut inputs = FxHashSet::default();
```

Comment:
I...thought we ripped this code out. Isn't this the old chalk integration code?

Reply:
Heh I can't tell.